caffe2/core/operator.h

```diff
@@ -192,6 +192,16 @@ class CAFFE2_API OperatorBase : public Observable<OperatorBase> {
     return XBlobGetMutableTensor(outputs_.at(idx), dims, options);
   }
 
+  void SetOutputTensor(int idx, Tensor tensor) {
```

Comment:
@jerryzh168 can you look at this and tell us if it makes sense? I feel that it might be problematic because the output for the operator may be preallocated, and we are expected to write into exactly that memory.

Reply:
I think it should be fine in terms of correctness since Tensor is a pointer now? user should be depending on the identity of Tensor instance right

Reply:
And it (hopefully) should be correct for perf. This is used in operator_c10wrapper.h (i.e. when calling a c10 op from caffe2 frontend) after calling the kernel to push the outputs back to the caffe2 frontend. However, before calling the kernel, operator_c10wrapper checks if there are any preallocated outputs and passes them to the kernel.

Reply:
is there still cost of conversion, wrapping and unwrapping tensors?